tests/unit/postTestBuildComment.ts

```diff
@@ -49,15 +54,18 @@ const message = `:test_tube::test_tube: Use the links below to test this adhoc b
 :eyes: [View the workflow run that generated this build](https://github.com/Expensify/App/actions/runs/1234) :eyes:
 `;
 
+const asMutable = <T>(value: T): Writable<T> => value as Writable<T>;
```

Comment:
I see you are creating this function in second PR , maybe its worth to move it to some utils folder?

Reply:
I agree, suggested the same in another PR